docs/python/environments.md

```diff
@@ -117,29 +77,19 @@ python3 -m venv .venv
 python -m venv .venv
 ```
 
-When you create a new virtual environment, a prompt will be displayed to allow you to select it for the workspace.
+When you create a new virtual environment, a prompt will be displayed to allow you to select it for the workspace. To learn more about the `venv` module, see [Creation of virtual environments](https://docs.python.org/3/library/venv.html) on Python.org.
 
 ![Python environment prompt](images/environments/python-environment-prompt.png)
 
-This will add the path to the Python interpreter from the new virtual environment to your workspace settings. That environment will then be used when installing packages and running code through the Python extension. For examples of using virtual environment in projects, see the [Django tutorial](/docs/python/tutorial-django.md) and the [Flask tutorial](/docs/python/tutorial-flask.md).
-
-If the activate command generates the message "Activate.ps1 is not digitally signed. You cannot run this script on the
-current system.", then you need to temporarily change the PowerShell execution policy to allow scripts to
-run (see [About Execution Policies](https://go.microsoft.com/fwlink/?LinkID=135170) in the PowerShell documentation):
-
-```cmd
-Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process
-```
+This will add the path to the Python interpreter from the new virtual environment to your workspace settings. That environment will then be used when installing packages and running code through the Python extension. For examples of using virtual environment in projects, see the [Python tutorial](/docs/python/python-tutorial.md), [Django tutorial](/docs/python/tutorial-django.md) and the [Flask tutorial](/docs/python/tutorial-flask.md).
 
 > **Note**: If you're using a version of the Python extension prior to 2018.10, and you create a virtual environment in a VS Code terminal, you must run the **Reload Window** command from the Command Palette and then use **Python: Select Interpreter** to activate the environment. If you have any problems with VS Code recognizing a virtual environment, please [file an issue](https://github.com/microsoft/vscode-docs/issues) in the documentation repository so we can help determine the cause.
```

Comment:
At this point I think we can remove this note too!

Reply:
Makes sense, removed.